test/test.js

```diff
@@ -103,30 +116,92 @@ test('relative', async (t) => {
 });
 
 test('integrityBlockSign', async (t) => {
-  const testPrivateKey = wbnSign.parsePemKey(
-    '-----BEGIN PRIVATE KEY-----\nMC4CAQAwBQYDK2VwBCIEIB8nP5PpWU7HiILHSfh5PYzb5GAcIfHZ+bw6tcd/LZXh\n-----END PRIVATE KEY-----'
-  );
-
   const signed = (
     await run({
-      baseURL:
-        'isolated-app://4tkrnsmftl4ggvvdkfth3piainqragus2qbhf7rlz2a3wo3rh4wqaaic/',
-      output: 'example.wbn',
+      baseURL: TEST_IWA_BASE_URL,
+      output: 'example.swbn',
       integrityBlockSign: {
-        key: testPrivateKey,
+        key: TEST_ED25519_PRIVATE_KEY,
       },
     })
   ).memfs;
-  t.deepEqual(signed.readdirSync('/out').sort(), ['example.wbn', 'main.js']);
+  t.deepEqual(signed.readdirSync('/out').sort(), ['example.swbn', 'main.js']);
 
-  const swbnFile = signed.readFileSync('/out/example.wbn');
+  const swbnFile = signed.readFileSync('/out/example.swbn');
   const wbnLength = Number(Buffer.from(swbnFile.slice(-8)).readBigUint64BE());
   t.truthy(wbnLength < swbnFile.length);
 
   const { signedWebBundle } = new wbnSign.IntegrityBlockSigner(
     swbnFile.slice(-wbnLength),
-    { key: testPrivateKey }
+    { key: TEST_ED25519_PRIVATE_KEY }
   ).sign();
-
   t.deepEqual(swbnFile, Buffer.from(signedWebBundle));
 });
+
+test('headerOverride - IWA with good headers', async (t) => {
+  const goodHeadersTestCases = [
+    iwaHeaderDefaults,
```

Comment:
Could you maybe also add sine extra headers, not just the iwa default headers, to verify that these extra headers also end up in the bundle?